py3dtilers/Common/tiler.py

```diff
@@ -114,6 +114,14 @@ def __init__(self):
                                  help='Set the maximum number of features in each tile when the features are distributed by a kd-tree.\
                                      The value must be an integer.')
 
+        self.parser.add_argument('--texture_lods',
```

Comment:
Since creating texture_lods seems to be always a good thing, do you think that we can create them by default if there is a texture? Pending that there is no texture quality parameter set by the user